src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.shared.cs

```diff
@@ -8,8 +8,8 @@ public partial class Snackbar : ISnackbar
 	static readonly WeakEventManager weakEventManager = new();
 
 	bool isDisposed;
-	string text = string.Empty;
-	string actionButtonText = AlertDefaults.ActionButtonText;
+	readonly string text = string.Empty;
+	readonly string actionButtonText = AlertDefaults.ActionButtonText;
```

Comment:
Cool!! I didn't know that we could use `readonly` fields in `get; init;` properties! 

I like it. It makes a lot of sense 💯